client/src/components/Button.svelte

```diff
@@ -29,49 +29,25 @@
     user-select: none;
     font-size: 0.9em;
     letter-spacing: 1px;
-    width: 12em;
-    height: 4em;
     text-align: center;
-    line-height: 4em;
     color: #ffffff;
     background-color: #000000;
     text-transform: uppercase;
     border-radius: 12px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    padding: 20px;
   }
-  .analytics-btn {
-    position: relative;
-    text-align: left;
-    padding-left: 20px;
-  }
-  .fa {
-    position: absolute;
-    right: 20px;
-    top: 35%;
-    transform: translateX(-50%);
-  }
+  
```

Comment:
![image](https://github.com/srm-kzilla/kzilla.xyz/assets/113087236/546eb770-9eb6-49fc-b4ed-880b9c24793e)

The size of these buttons are changing on clicking. I think it's because you're deleting classes that are being used in other places.